Source/WebCore/rendering/RenderObject.h

```diff
@@ -1144,27 +1153,24 @@ class RenderObject : public CachedImageClient, public CanMakeCheckedPtr {
         void set##Name(Type name) { m_##name = static_cast<unsigned>(name); }\
 
     enum class RenderObjectFlag : uint32_t {
-        IsAnonymous = 1 << 0,
-        IsRenderText = 1 << 1,
-        IsRenderBox = 1 << 2,
-        IsBlock = 1 << 3,
-        IsReplacedOrInlineBlock = 1 << 4,
-        BeingDestroyed = 1 << 5,
-        NeedsLayout = 1 << 6,
-        NeedsPositionedMovementLayout = 1 << 7,
-        NormalChildNeedsLayout = 1 << 8,
-        PosChildNeedsLayout = 1 << 9,
-        NeedsSimplifiedNormalFlowLayout = 1 << 10,
-        EverHadLayout = 1 << 11,
-        IsExcludedFromNormalLayout = 1 << 12,
-        Floating = 1 << 13,
-        VerticalWritingMode = 1 << 14,
-        PreferredLogicalWidthsDirty = 1 << 15,
-        HasRareData = 1 << 16,
-        HasLayer = 1 << 17,
-        HasNonVisibleOverflow = 1 << 18,
-        HasTransformRelatedProperty = 1 << 19,
-        ChildrenInline = 1 << 20,
+        IsBlock = 1 << 0,
```

Comment:
Does this represent the same state as one of the TypeFlags?

Reply:
No, this one represents that this object is `display: block` vs inline (i.e. state) whereas `IsRenderBlock` is about the type.

Reply:
My plan is to rename `RenderObjectFlag` to `StateFlag` so that this distinction is more clear.